src/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "sajari/sajari-sdk-php",
+  "description": "Sajari SDK for PHP",
+  "minimum-stability": "dev",
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
@tbillington is this Github repo not on Packagist.org? Why does it need a custom `repositories` definition?